docs/traceflow-guide.md

```diff
@@ -0,0 +1,103 @@
+# Traceflow User Guide
+
+Antrea supports using Traceflow for network diagnosis: it generates tracing requests for traffic going through
+Antrea-managed Pod network. Creating a new Traceflow CRD triggers the Traceflow module to inject packet into OVS,
+provide various observation points along the packet's path and populate these observations into the status field of
+the Traceflow CRD. Users can start a new trace simply from either Kubectl or Antrea-Octant-Plugin and view Traceflow
+result via CRD or UI graph. We will also provide a corresponding Antctl to start a new trace in the near future.
+
+## Table of Contents
+
+- [Prerequisites](#Prerequisites)
+- [Start a New Trace](#Start-a-New-Trace)
+  - [Using kubectl and YAML file](#using-kubectl-and-YAML-file)
+  - [Using Octant with antrea-octant-plugin](#Using-Octant-with-antrea-octant-plugin)
+- [View Traceflow Result and Graph](#View-Traceflow-Result-and-Graph)
+- [View Traceflow CRDs](#View-Traceflow-CRDs)
+
+## Prerequisites
+You need to switch on traceflow from featureGates defined in antrea.yml for both Controller and Agent.
+```yaml
+  antrea-controller.conf: |
+    featureGates:
+    # Enable traceflow which provides packet tracing feature to diagnose network issue.
+      Traceflow: true
+  antrea-agent.conf: |
+    featureGates:
+    # Enable traceflow which provides packet tracing feature to diagnose network issue.
+      Traceflow: true
+```
+For antrea-octant-plugin installation, please refer to [antrea-octant-installation](/docs/octant-plugin-installation.md).
+
+## Start a New Trace
+You can choose to use either Kubectl together with YAML file or Octant UI to start a new trace.
+If you use Kubectl to start a new trace, we now support traceflow from Pod to either Pod or IP and
+you can also mark the protocol(TCP/UDP/ICMP) and desired ports of the trace. 
+If you use UI to start a new trace, we currently only support destination to be Pod, but will support
+destination to be IP or Service soon.
+
+### Using kubectl and YAML file
+You can start a new trace by creating Traceflow CRD via Kubectl and a YAML file which contains the essential
+configuration of Traceflow CRD. An example YAML file of Traceflow CRD might look like this:
+```yaml
+apiVersion: ops.antrea.tanzu.vmware.com/v1alpha1
+kind: Traceflow
+metadata:
+  name: tf-test
+spec:
+  source:
+    namespace: default
+    pod: tcp-sts-0
+  destination:
+    namespace: default
+    pod: tcp-sts-2
+#   ip: IP can also be marked as destination, but namespace/pod and ip are mutually exclusive.
+  packet:
+    ipHeader:
+      protocol: 6 # Protocol here can be 6 (TCP), 17 (UDP) or 1 (ICMP), default value is 1 (ICMP)
```

Comment:
Not related to this PR. This reminds me we probably should use string "TCP", "UDP", "ICMP" for protocols like NetworkPolicy does to make it more friendly, otherwise people like me won't be able to set correct protocol without googling once every time. @jianjuns @antoninbas @gran-vmv what do you think?
https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L619

Reply:
+1 - but is breaking backwards-compatibility an issue? maybe open an issue and we will make that change when we switch the API to beta, along with any other change that may come up until then?